CONTRIBUTING.md

```diff
@@ -9,7 +9,7 @@ Cisco Network Elements support a rich set of features to make networks robust, e
 ## Making Changes
 
 * Fork and clone the repository
-  * Run the `bin/git/update-hooks` script to install our recommended Git hooks into your local repository.
+  * Run the `bin/git/update-hooks` script to install our recommended Git hooks into your local repository.  (Note: some of the hooks require additional tools, such as the `rubocop` ruby gem, be installed and available in your `$PATH`.)
```

Comment:
> On Dec 22, 2015, at 2:36 PM, Rob Gries <notifications@github.com> wrote:
> Adding a pointer to our Ruby Prerequisites <https://github.com/jamtaylo/cisco-network-node-utils/blob/getting-started-docs/docs/README-develop-node-utils-APIs.md#ruby-prerequisites> might be helpful. I'm not sure what additional tools are needed other than rubocop.
> 
That's the only one I had to install on top of the defaults that come with OS X.